scripts/localstack/init/localstack_init.sh

```diff
@@ -8,6 +8,14 @@ awslocal dynamodb create-table \
     AttributeName=id,KeyType=HASH \
   --provisioned-throughput ReadCapacityUnits=1000,WriteCapacityUnits=1000
 
+awslocal dynamodb update-table \
+    --table-name identity-verify \
+    --attribute-definitions AttributeName=lpas,AttributeType=SS \
+    --global-secondary-index-updates \
+        "[{\"Create\":{\"IndexName\": \"lpas-index\",\"KeySchema\":[{\"AttributeName\":\"lpas\",\"KeyType\":\"HASH\"}], \
+        \"ProvisionedThroughput\": {\"ReadCapacityUnits\": 20, \"WriteCapacityUnits\": 10 }, \
+        \"Projection\":{\"ProjectionType\":\"ALL\"}}}]"
```

Comment:
Is this index needed? I couldn't see it being used.

If it is needed, you'll also need to add it to [the infrastructure code](https://github.com/ministryofjustice/opg-sirius-infrastructure/blob/8ace6e6a5998a281c62aef5cddf8c808816c337c/environment/modules/lpa_identity_dynamodb/dynamodb.tf).